scripts/tools/StatementsMigrationWrapper.php

```diff
@@ -0,0 +1,240 @@
+                'defaultValue' => 10000,
+                'description' => 'Amount of `statements` rows provided into taskqueue to proceeded with'
+            ],
+            'pickSize' => [
+                'prefix' => 'p',
+                'longPrefix' => 'pickSize',
+                'required' => false,
+                'cast' => 'integer',
+                'defaultValue' => 0,
+                'description' => 'Amount of items proceed in chunk (for test purposes)'
+            ],
+            'recoveryMode' => [
+                'prefix' => 'r',
+                'longPrefix' => 'recoveryMode',
+                'required' => false,
+                'cast' => 'boolean',
+                'defaultValue' => false,
+                'description' => 'Starts recovery by resuming from the last chunk'
+            ],
+            'repeat' => [
+                'prefix' => 'rp',
+                'longPrefix' => 'repeat',
+                'required' => false,
+                'cast' => 'boolean',
+                'defaultValue' => true,
+                'description' => 'Scan all the records to the very end'
+            ],
+            'start' => [
+                'prefix' => 's',
+                'longPrefix' => 'start',
+                'required' => false,
+                'cast' => 'integer',
+                'defaultValue' => 1,
+                'description' => 'Sliding window start range'
+            ],
+            'queue' => [
+                'prefix' => 'q',
+                'longPrefix' => 'queue',
+                'required' => false,
+                'cast' => 'string',
+                'description' => 'Define task queue broker name to work at'
+            ],
+
+            'target' => [
+                'prefix' => 't',
+                'longPrefix' => 'target',
+                'required' => true,
+                'cast' => 'string',
+                'description' => sprintf(
+                    'Define unit processing task (must extend "%s")',
+                    AbstractStatementMigrationTask::class
+                )
+            ],
+        ];
+    }
+
+    protected function provideDescription()
+    {
+    }
+
+    protected function run()
+    {
+        $startedAt = time();
+
+        $chunkSize = $this->getOption('chunkSize');
+        $start = $this->getOption('start');
+        $isRecovery = $this->getOption('recoveryMode');
+        $pickSize = $this->getOption('pickSize');
+        $repeat = $this->getOption('repeat');
+        $queue = $this->getOption('queue');
+
+        $taskClass = $this->detectTargetClass($this->getOption('target'));
+
+        if ($queue) {
+            $report = $this->configureQueues($queue, $taskClass);
+            $result = common_report_Report::createSuccess(
+                sprintf(
+                    '1. Please restart fpm to apply changes ' . PHP_EOL .
+                    '2. Execute `php index.php "oat\taoTaskQueue\scripts\tools\RunWorker" --queue=%s`' . PHP_EOL .
```

Comment:
```suggestion
                    '2. Execute `php index.php "' . RunWorker::class . '" --queue=%s`' . PHP_EOL .
```

So if class changes you can track usages here.